Umbraco-Heartcore/API-Documentation/Content-Management/forms/index.md

```diff
@@ -0,0 +1,305 @@
+            "href": "/api/forms/2edaf583-cf66-4d57-930c-f0772c3d1c52"
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+## Get by ID
+
+Get a specific form by its ID.
+
+**URL**: `/forms/{id}`
+
+**Method**: `GET`
+
+**Permissions required** : Access to Forms section of the Umbraco Backoffice
+
+### Success Response
+
+**Code**: 200
+
+**Content Example**:
+
+```json
+{
+  "_id": "2edaf583-cf66-4d57-930c-f0772c3d1c52",
+  "indicator": "*",
+  "name": "Contact",
+  "nextLabel": "Next",
+  "previousLabel": "Previous",
+  "submitLabel": "Submit",
+  "disableDefaultStylesheet": false,
+  "fieldIndicationType": "MARK_MANDATORY_FIELDS",
+  "hideFieldValidation": false,
+  "messageOnSubmit": "Thank you",
+  "showValidationSummary": false,
+  "pages": [
+    {
+      "fieldsets": [
+        {
+          "columns": [
+            {
+              "width": 12,
+              "fields": [
+                {
+                  "caption": "Name",
+                  "alias": "name",
+                  "required": true,
+                  "requiredErrorMessage": "Please provide a value for Name",
+                  "settings": {
+                    "placeholder": "John Smith",
+                    "defaultValue": ""
+                  },
+                  "type": "text"
+                },
+                {
+                  "caption": "Email",
+                  "alias": "email",
+                  "required": true,
+                  "requiredErrorMessage": "Please provide a value for Email",
+                  "settings": {
+                    "placeholder": "johnsmith@example.org",
+                    "defaultValue": "",
+                    "pattern": "[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+.[a-zA-Z0-9-.]+",
+                    "patternInvalidErrorMessage": "Please enter a valid email address"
+                  },
+                  "type": "text"
+                },
+                {
+                  "caption": "Message",
+                  "alias": "message",
+                  "required": false,
+                  "requiredErrorMessage": "Please provide a value for Message",
+                  "settings": {
+                    "defaultValue": "",
+                    "placeholder": ""
+                  },
+                  "type": "textarea"
+                },
+                {
+                  "caption": "Consent for storing submitted data",
+                  "alias": "dataConsent",
+                  "required": true,
+                  "requiredErrorMessage": "Consent is required to store and process the data in this form.",
+                  "settings": {
+                    "acceptCopy": "Yes, I give permission to store and process my data"
+                  },
+                  "type": "dataConsent"
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "_links": {
+    "self": {
+      "href": "/api/forms/2edaf583-cf66-4d57-930c-f0772c3d1c52"
+    }
+  }
+}
+```
+
+## Submit entry
+
```

Comment:
Missing some intro/summary type text like:

Submit form entries for a specific Form - this is what you would use when a form is submitted from your presentation layer.

Reply:
I've added the suggested text